lib/ruby_lsp/requests/support/sorbet.rb

```diff
@@ -8,36 +8,41 @@ class Sorbet
         class << self
           extend T::Sig
 
+          class Annotation < T::Struct
+            prop :arity, T.any(Integer, T::Range[Integer])
+            prop :receiver, T.nilable(T::Boolean)
+          end
```

Comment:
We moved away from using `T::Struct`s for performance reasons in a past PR. Can we make this just a regular Ruby object instead?

Also, just a nitpick, I'd move the class outside of the singleton class block.

Reply:
Updated. Would it be useful to have a cop to warn about `T::Struct`?